app/data/ttbarbits-12evt-118prim.hepmc3

```diff
@@ -2,93 +2,93 @@ HepMC::Version 3.02.05
 HepMC::Asciiv3-START_EVENT_LISTING
 E 1 0 10
 U GEV MM
-P 1 0 -24 -4.7100161999999997e+01 -1.2502119999999999e+02 -2.2036460000000000e+02 2.6993799103269066e+02 8.0356931000000003e+01 2
-P 2 0 24 1.5931562000000000e+02 -2.8732734000000001e+01 -2.1139886999999999e+02 2.7794694522156675e+02 7.9736975999999999e+01 2
-P 3 0 -24 -4.5609923999999999e+01 -1.1829483999999999e+02 -2.0798566000000000e+02 2.5649406516191704e+02 8.0356931000000003e+01 2
-P 4 0 24 1.5891660999999999e+02 -2.8659172999999999e+01 -2.1094170000000000e+02 2.7736298107615335e+02 7.9736975999999999e+01 2
+P 1 0 -24 -4.7100161999999997e+01 -1.2502119999999999e+02 -2.2036460000000000e+02 2.6993799103269066e+02 8.0356931000000003e+01 1
+P 2 0 24 1.5931562000000000e+02 -2.8732734000000001e+01 -2.1139886999999999e+02 2.7794694522156675e+02 7.9736975999999999e+01 1
```

Comment:
I guess none of these had an outgoing vertex? Did we generate this file?

Reply:
I made this a long time ago by taking some Pythia output file (which I think @stognini generated?) and trimmed out everything but the EM tracks... it's basically garbage. It lets the tests "run" ... if we want them to be meaningful we have to change a lot more than just the input file.